### ISSUE TITLE
Gentoo zlib compilation issue

### ISSUE THREAD

**Issue by jason-oliveira** [yvt/openspades#423]
Gentoo, in an effort to [clean out their namespace](https://bugs.gentoo.org/show_bug.cgi?id=383179), deleted zlib's OF. This makes the app break.

Sources/unzip/ioapi.h needs to be modified, in order to add the following lines:

```
/* Gentoo removed OF from their copy of zconf.h (https://bugs.gentoo.org/show_bug.cgi?id=383179) */
/* but our copy of minizip needs it. */
#ifndef OF
#define OF(args) args
#endif
```

around line 21 or so.

**Comment by yvt** [comment]
PR?

**Comment by feikname** [comment]
Fixed in #437
